src/posthog-core.js

```diff
@@ -218,11 +212,7 @@ PostHogLib.prototype.init = function (token, config, name) {
         return
     }
 
-    var instance = create_mplib(token, config, name)
-    posthog_master[name] = instance
```

Comment:
readd this @macobo

Reply:
Is there a test that would have failed?

Reply:
Sadly nope - the library init code is full of gremlins.